File: app/src/main/java/com/myapp/modmenu/MainActivity.java
package com.myapp.modmenu;

import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;
import android.view.Gravity;
import android.graphics.Color;

public class MainActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        System.loadLibrary("mymod");

        TextView tv = new TextView(this);
        tv.setText("MyModMenu Loaded\n\nLaunch LAC to activate.");
        tv.setGravity(Gravity.CENTER);
        tv.setTextColor(Color.WHITE);
        tv.setBackgroundColor(Color.BLACK);

        setContentView(tv);
    }
}

File: app/src/main/java/com/myapp/modmenu/MainActivity.kt
package com.myapp.modmenu

import android.app.Activity
import android.os.Bundle
import android.widget.TextView
import android.view.Gravity

class MainActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        System.loadLibrary("mymod")

        val tv = TextView(this)
        tv.text = "MyModMenu Loaded\n\nCheck logcat:\nadb logcat -s MyMod"
        tv.gravity = Gravity.CENTER
        setContentView(tv)
    }
}
